src/components/Card.module.css

```diff
@@ -0,0 +1,29 @@
+/* .card {
+  border-radius: 12px;
+  box-shadow: 0 1px 8px rgba(0, 0, 0, 0.25);
+  width: 200px;
+  height: auto;
+  margin: 5px;
+}
+
+.text_content {
+  padding: 20px;
+} */
+
+/* spostati in container.module.css!! */

```

Comment:
Puoi cancellare questo commento.
Se vuoi mantenere un reminder qui, scrivi pure un commento dicendo in dettaglio che lo styling the container che contiene la carta è stato spostato nel suo relativo component.